.travis.yml

```diff
@@ -13,7 +13,7 @@ before_install:
 
 script:
   - go vet ./...
-  - golint -set_exit_status `go list ./... | grep -v /vendor/`
+  - golint -set_exit_status `go list ./... | grep -Ev "(stackint/asm|vendor)"`
```

Comment:
The stackint/asm package contains code from `math/big` which contains function names with underscores, which golint doesn't like.